submit-queue/pod.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: submit-queue
+  namespace: default
+spec:
+  containers:
+  - image: gcr.io/google_containers/submit-queue:0.1
+    command:
+      - bash
+      - -c
+      - "/submit-queue --user-whitelist=/whitelist.txt --jenkins-host=http://jenkins-master:8079 --token=$(cat /etc/secret-volume/token) --alsologtostderr"
```

Comment:
That's the name of the VM that the jenkins instance runs on.  The cluster runs in the same GCE project, so it just uses the built in project DNS.